src/components/ErrorBoundary/ErrorBoundary.styles.js

```diff
@@ -0,0 +1,44 @@
+import styled from '@emotion/styled';
+
+export const Content = styled.main`
+  align-items: center;
+  background-color: ${({ theme }) => theme.color.gray100};
+  color: ${({ theme }) => theme.color.gray800};
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  min-height: 100vh;
+  padding: 1.5rem;
+  text-align: center;
+`;
+
+export const Wrapper = styled.div`
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+`;
+
+export const Legend = styled.p`
+  font-size: 1.875rem;
+`;
+
+export const ErrorMessage = styled.p`
+  margin-top: 1.25rem;
+  margin-bottom: 1.25rem;
+  width: 50%;
+  text-align: left;
+`;
+
+export const BtnReportIssue = styled.a`
+  background-color: ${({ theme }) => theme.color.indigo600};
+  border-radius: 9999px;
```

Comment:
maybe we can use the new style of a not fully rounded button that is used on other screens